backend/collaboration-service/app/src/helpers/rooms.helper.ts

```diff
@@ -1,12 +1,18 @@
 import { Text } from "@codemirror/state"
 import moment from "moment"
+import { DIFFICULTY, TOPIC_TAG } from "../constants/question"
+import { DEFAULT_LANGUAGE } from "../constants/language"
 
-export const generateNewDocument = (roomId: string) => {
+export const generateNewDocument = (roomId: string, difficulty: keyof typeof DIFFICULTY, topic: keyof typeof TOPIC_TAG) => {
     return {
         created: moment(),
         updated: moment(),
         updates: [],
         doc: Text.of([`Welcome to Room ${roomId}`]),
-        pending: []
+        pending: [],
+        difficulty,
+        language: DEFAULT_LANGUAGE,
+        users: [],
```

Comment:
i thought we were gna do the totalConnectedUsers and currentConnectedUsers thign

Reply:
As discussed privately, this is not the full implementation but eventually i will add a users object with a "connected" attribute to check if the user is connected